general/pair.js

```diff
@@ -178,8 +178,16 @@ module.exports = {
     makeBatchRequest: async function (w3, requests) {
         let batch = new w3.BatchRequest();
 
-        requests.forEach((request) => batch.add(request.req))
+        requests.forEach((request) => {
+            batch.add(request.req)
+                .catch(err => {
+                    console.log(`Batch Request: ${err.message}`);
```

Comment:
We should throw an error here.
Ignoring the errors and just logging a message can provide wrong results,

Reply:
now it returns error message,
the error message is catched and returned to the user as response of request. example:

```
{

      "success": false,
      "appId": "4235674645294935513828530749343108731671539155305792113046250148823270483942",
      "error": {
            "message": "Batch request failed: Batch request timeout",
            "onRemoteSide": true
      }

}
```